rubicon_ml/cli.py

```diff
@@ -78,7 +82,75 @@ def ui(root_dir, page_size, project_name, host, port, debug, storage_options):
     dashboard.serve(run_server_kwargs=run_server_kwargs)
 
 
-# CLI groups
+@cli.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+    )
+)
+@click.option(
+    "--root-dir",
+    type=click.STRING,
+    default=os.environ.get("RUBICON_ROOT_DIR"),
+    help="The absolute path to the top level folder holding the rubicon-ml project.",
+    required=False,
+)
+@click.option(
+    "--project-name",
+    default=os.environ.get("RUBICON_PROJECT_NAME"),
+    type=click.STRING,
+    help="Name of rubicon project to retrieve experiments from",
+    required=False,
+)
+@click.option(
+    "--color",
+    "-c",
+    type=click.STRING,
+    help="Color of query output",
+    required=False,
+)
+@click.option(
+    "--bf",
+    is_flag=True,
+    show_default=True,
+    default=True,
+    required=False,
+    help="Flag that switches pretty print output format to basic format",
+)
+@click.argument(
+    "query",
+    nargs=1,
+    required=True,
+)
+def search(root_dir, project_name, color, bf, query):
+    """
+    This function provides capability to query  rubicon experiments from the command line without using python
+    directly and output the JSON in the command line.
```

Comment:
can we get doc strings for the parameters? Or `bf` specifically

Reply:
they're in the definitions of each of the `click.options` - you can view the full help message with `rubicon_ml search --help`